oval_graph/client.py

```diff
@@ -35,6 +35,7 @@ def __init__(self, args):
         self.START_OF_FILE_NAME = 'graph-of-'
         self.date = str(datetime.now().strftime("-%d_%m_%Y-%H_%M_%S"))
         self.verbose = self.arg.verbose
+        self.html_builder = None
```

Comment:
Why not to do `self.html_builder = BuilderHtmlGraph(self.parts, self.verbose)` here? We clearly need a `self.html_builder` in every `Client` instance.